exampleCourse/questions/demoAutograderNumpy/question.html

```diff
@@ -0,0 +1,22 @@
+<pl-question-panel>
+  <p>
+    This question uses the built-in methods in <code>code_feedback.py</code> to check NumPy arrays. In addition, specific library functions can be disabled by hooking them in the setup autograder code.
```

Comment:
Should we be styling these "comment" parts of the question differently from the rest of the question? Perhaps as italics? Do we have a standard for this already?

Reply:
Some other elements are enclosing the actual question portion with a card, but I think that looks strange here because there's only one "question".  I'll go with italics.